### PR TITLE
terravaloper1vqnhgc6d0jyggtytzqrnsc40r4zez6tx99382w

### PR DESCRIPTION
**Pull request by fish2plain**
edit kysenpool-profile

### CODE REVIEW

File: validators/terravaloper1vqnhgc6d0jyggtytzqrnsc40r4zez6tx99382w/profile.json [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {
-  "contact": {
-    "email": "",
-    "telegram": ""
+  "contact": { 
+      "email": "devops@kysenpool.io",
+      "telegram": "https://t.me/KysenPool",
+      "tweeter":  "https://twitter.com/kysenpool"
```
Comment (Jared-Warn) [inline validators/terravaloper1vqnhgc6d0jyggtytzqrnsc40r4zez6tx99382w/profile.json:5]: change from "tweeter" to "twitter"

File: validators/terravaloper1vqnhgc6d0jyggtytzqrnsc40r4zez6tx99382w/profile.json [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+  },
+  "notifications": {
+      "email": "devops@kysenpool.io" 
   }
 }
```
Comment (Jared-Warn) [inline validators/terravaloper1vqnhgc6d0jyggtytzqrnsc40r4zez6tx99382w/profile.json:10]: remove extra bracket

Comment (fish2plain) [reply validators/terravaloper1vqnhgc6d0jyggtytzqrnsc40r4zez6tx99382w/profile.json:10]: sorry, can't tell which bracket is extra

Comment (gregnuj) [reply validators/terravaloper1vqnhgc6d0jyggtytzqrnsc40r4zez6tx99382w/profile.json:10]: It looks like Github complaining there is not a newline at the end of the file (not an extra bracket), add newline to the end of profile.json

**Pull request merged**